docs/source/config.rst

```diff
@@ -0,0 +1,33 @@
+Configuration
+=============
+
+MASH Client uses a YAML config file. By default the client will look
```

Comment:
s/config/configuration/

"Which is expected with the path *~/.config/mash_client/config.yaml*"